ddtrace/internal/processor/truncator.py

```diff
@@ -0,0 +1,77 @@
+"""
+From agent truncators: https://github.com/DataDog/datadog-agent/blob/main/pkg/trace/agent/truncator.go
+"""
+
+from . import SpanProcessor
+
+
+# Values from:
+# https://github.com/DataDog/datadog-agent/blob/main/pkg/trace/traceutil/truncate.go#L22-L27
```

Comment:
Can we use a commit hash here, code changes over time

Reply:
```suggestion
# https://github.com/DataDog/datadog-agent/blob/ff91f174c6534be2b6f24baddc71eaec9224b0fc/pkg/trace/traceutil/truncate.go#L22-L27
```